popsborder/consignments.py

```diff
@@ -355,6 +416,7 @@ def get_consignment_generator(config):
         )
     else:
         raise RuntimeError(
-            f"Unknown consignment generation method: {generation_method}"
+            f"Unknown consignment generation method: {generation_method} or "
+            f"input file type: {config['input_file']['file_type']}"
```

Comment:
This will fail for unknown `generation_method` (e.g., `param_based`) and `config['input_file']['file_type']` not provided. One way around it would be to extract `config["input_file"]["file_type"]` ahead of time if available and use None otherwise. This will make the code more robust overall because the case of `generation_method == "input_file"` and forgotten `config["input_file"]["file_type"]` is not handled at all.